lute/cli/src/climain.cpp

```diff
@@ -354,10 +358,11 @@ static std::optional<std::string> getValidPath(std::string filePath)
         fallbackPath += ext;
 
         if (isFile(fallbackPath))
-            return fallbackPath;
+            return {true, fallbackPath};
     }
 
-    return std::nullopt;
+
+    return {false, std::nullopt};
```

Comment:
Same in this function. We can just return `std::pair<bool, std::string>` to avoid the caller  needing to repeat the "file not found" message. (Also, maybe naming this `res` or something like that makes more sense than `path`, to be more similar to `pcall`'s interface and make a bit more sense in the erroring case.)

```suggestion
    return {false, path};
```